charts/airbyte/Chart.yaml

```diff
@@ -15,13 +16,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.7
+version: 0.39.36
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.39.37-alpha"
+appVersion: "0.39.23-alpha"
```

Comment:
shouldn't this be the same as the version on line 19?

Reply:
Let me fix that